src/Public/js/nipsweb.librarypane.js

```diff
@@ -1,52 +1,75 @@
-/* global Bloodhound, myradio, planner, ulsort */
+/* global Bloodhound, myradio, planner, ulsort, showAlert */
 /**
  * This file contains the necessary functions for browsing searching and adding
  * items in the Library pane of the main interface
  * @todo Includes length attribute to list items
  */
 
+//Still can't believe js/jquery doesn't have escaping built-in!
```

Comment:
Maybe I'm using the wrong term, I mean converting into html special characters like &amp; from &, since the tooltips won't work with ' in the titles (closes attribute and keeps going).

There tended to be quite a bit of confusion among the interwebs with this one, and I've been directed to that link before. Using that just gives me %23 etc within the displayed tooltip.